make/Docs.gmk

```diff
@@ -332,6 +332,9 @@ define SetupApiDocsGenerationBody
   $$(eval $$(call create_overview_file,$1))
   $1_OPTIONS += -overview $$($1_OVERVIEW)
 
+  # Add summary pages for new/deprecated APIs in recent releases
+  $1_OPTIONS += --since 12,13,14,15,16,17 --since-label "New API since JDK 11"
```

Comment:
How do you expect this to change as the JDK version is bumped? If there is an expected pattern, then maybe we should try to generate the list instead, so we don't need to manually update twice a year.

Reply:
I don't have a solution for this problem. AFAIK there are a few things that have to be "bumped" manually between releases, and this would be yet another. Obviously not a great solution.

Reply:
Yes, there are a few things, but in the build itself, we are down to a single config file today, so I would really appreciate if this could be figured out. I can provide the implementation for generating this, but I need to understand what the expected pattern is. From what I can see, it looks like $(sequence N, M) where N is the last LTS+1 and M is current JDK version. Then the string has "JDK N" in it. M is already well defined, so the only new input here is N, which we could move to the version numbers config file (make/conf/version-numbers.conf). Something like DEFAULT_VERSION_DOCS_API_SINCE=11. There is some additional boilerplate needed, and the sequence macro of course, but does this sound reasonable to you?

Reply:
We actually have a sequence macro already, so generating the list can be done like this:

$(call CommaList,$(call sequence 12, 17))

Or if what you have is 11 and 17:

$(call CommaList,$(filter-out 11, $(call sequence 11, 17)))

Reply:
Thanks for the help and for providing all the pieces, Erik! I just added a new commit and I think I got everything working.